tools/base/pytest_utils.py

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
Isn't this the same as `pytest_runner.py`?

Reply:
Do we need one of these files for everything we use `pytest` with?

Reply:
atm yes - one per target

this is defo the bit i would change with this implementation - v open to suggestions on how to do it without

Reply:
updated with suggestion from @htuch